extensions/amp-accordion/1.0/animations.js

```diff
@@ -124,12 +109,14 @@ export function animateCollapse(content) {
  * @return {!UnlistenDef}
  */
 function animate(element, prepare, cleanup = undefined) {
+  element.classList.add('i-amphtml-animating');
   let player = prepare();
   player.onfinish = player.oncancel = () => {
     player = null;
     if (cleanup) {
       cleanup();
     }
+    element.classList.remove('i-amphtml-animating');
```

Comment:
We also need to put this in `player.oncancel`.

Reply:
As discussed offline, in this case onfinish and oncancel should both do the same and should both remove the class at the end! :)